### DISCUSSION TITLE
SBAS Atmospheric Correction

### DISCUSSION THREAD

**Discussion by krkrueger** [gmtsar/gmtsar#757] [Q&A]
Hey All,

I used the -atm 3 option when doing SBAS to get my aps grids.  My intention was going to be to then subtract the atmospheric noise components directly from the interferograms. For example:

intfT1_T2_corrected = intfT1_T2-apsT2+apsT1

I have an area that has deformation, that shows up in intfT1_T2.  The problem I am having is apsT1 is showing a significant amount of the actual deformation in it.  So when I make this atmospheric correction, it is shifting my deformation 1 or 2 interferograms earlier than it shows up in the non corrected images.

So I am assuming this method is not sufficient, should I be using different values for the -atm option in SBAS? should I filter the atmospheric grids to try and get rid of the actual deformation signal? or should I just not use the aps.grds at all on the individual interferograms, why would the deformation signal still be present?

Thanks!

**Comment by Xiaohua-Eric-Xu** [comment]
> The problem I am having is apsT1 is showing a significant amount of the actual deformation in it.

Do note the Common Scene Stacking approach has its limitations, especially when you have large, non-linear deformation, it will leak into APS estimates. Also, CSS methods have inadequate constraints at the beginning and end of the time-series. The underlying assumption is slowly changing deformation and will likely fail with transient signals (e.g., linear+seasonal+long-term-changes).

Tymofyeyeva, E., & Fialko, Y. (2015). Mitigation of atmospheric phase delays in InSAR data, with application to the eastern California shear zone. Journal of Geophysical Research: Solid Earth, 120(8), 5952-5963.

**Reply by krkrueger** [reply]
Ok, thanks for that, yeah the signal can be 2-3 cm during 1 12 day S1 image period, so fairly large, and very quick.  Makes sense now why this wouldn't work.  I went really heavy handed with a median filter on the APS estimates as a test to try and remove any of this signal leakage, we will see if that works.  If not I might have to try one of the detrending options. Thanks again!
